src/yb/yql/pgwrapper/pg_libpq-test.cc

```diff
@@ -190,6 +190,36 @@ TEST_F(PgLibPqTest, Simple) {
   ASSERT_EQ(row, (decltype(row){1, "hello"}));
 }
 
+TEST_F(PgLibPqTest, PgStatIdxScanNoIncrementOnErrorTest) {
+  auto conn = ASSERT_RESULT(Connect());
+
+  std::string createTableStmt = "create table many (";
```

Comment:
```suggestion
  std::string createTableStmt = "CREATE TABLE many (";
```

use proper capitalization in your SQL